packages/CAM_DIAG/amwg_template.csh

```diff
@@ -1,4 +1,4 @@
-#!/bin/csh
+#!/bin/csh -e
```

Comment:
Usually, we want to keep the scrip running even if there are errors.
So I would suggest to remove '-e' option.